### PR TITLE
Fix credential file updating

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #3501.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open your profile credential list
- click edit on a GCE cred
- choose a new file
- submit the form
- the form should close

**Comment by huwshimi** [comment]
[Fix updating creds](https://trello.com/c/g47NTIJP/312-fix-updating-creds)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8501/

File: jujugui/static/gui/src/app/components/profile/credential-list/credential-list.js [hunk 1]
```diff
@@ -178,13 +178,18 @@ class ProfileCredentialList extends React.Component {
 
   /**
     Generates the edit credential UI elements.
-    @param {Object} credential The credential details of the credential being
+    @param {Object} credentialName The name of the credential being
+      edited.
+    @param {Object} credentialName The id of the credential being
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/profile/credential-list/credential-list.js]: these keys are incorrect

**Comment by mitechie** [comment]
QA is ok, I can now update and add a new GCE credential.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8502/

**Comment by huwshimi** [comment]
Thanks @mitechie and @hatched

**Comment by huwshimi** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2696

**Comment by huwshimi** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**